models/model.js

```diff
@@ -37,6 +37,17 @@ exports.fetchUsers = () => {
 
 exports.fetchArticles = () => {
     return db.query("SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, CAST(COUNT(comments.article_id) AS INT) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC")
-
     .then((result) => result.rows)
+}
+
+exports.fetchArticleComments = (article_id) => {
+    if (/[^\d]/gi.test(article_id)) {
+        return Promise.reject(({status: 400, msg: "Invalid article_id"}))
```

Comment:
If the article id is anything other than a number, PSQL would throw its own error.  This error would be caught by the catch block in your controller and send to your error handling middleware with a code (22P02 - I think)